mtp_noms_ops/apps/user_admin/views.py

```diff
@@ -0,0 +1,17 @@
+from django.utils.translation import gettext_lazy as _
+from mtp_common.user_admin.views import UserCreationView as UserCreationViewBase
+
+from .forms import UserUpdateForm
```

Comment:
Any reason why you've used a relative import here, but absolute imports elsewhere?

Reply:
You mean like `from user_admin.forms import` instead of `from user_admin.forms import`?

I think I saw this in other places in the codebase and imported the same way here, it makes sense as I'm importing from within the same module, hence the `from .forms`.